crates/polars-mem-engine/src/executors/scan/ndjson.rs

```diff
@@ -40,6 +40,18 @@ impl JsonExec {
 
         let mut n_rows = self.file_scan_options.n_rows;
 
+        // Avoid panicking
+        if n_rows == Some(0) {
```

Comment:
drive-by - fix panic on `scan_ndjson().head(0)`

Reply:
Can you add a test for this one?

Reply:
added a test below